jsonrpsee/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "jsonrpsee"
+description = "JSON-RPC crate"
+version = "0.2.0-alpha.4"
+authors = ["Parity Technologies <admin@parity.io>", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+http-client = { path = "../http-client", version = "0.2.0-alpha.4", package = "jsonrpsee-http-client" }
+http-server = { path = "../http-server", version = "0.2.0-alpha.4", package = "jsonrpsee-http-server" }
+ws-client = { path = "../ws-client", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-client" }
+ws-server = { path = "../ws-server", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-server" }
+types = { path = "../types", version = "0.2.0-alpha.4", package = "jsonrpsee-types" }
+utils = { path = "../utils", version = "0.2.0-alpha.4", package = "jsonrpsee-utils" }
+proc-macros = { path = "../proc-macros", version = "0.2.0-alpha.4", package = "jsonrpsee-proc-macros" }
```

Comment:
Should this be renamed to `jsonrpsee-derive` perhaps?

Reply:
I think the `proc-macros` is more accurate because my intuition is that `derive` implies `custom derive macro` but maybe it's just me ^^

I don't have strong opinions on it, //cc @maciejhirsz ^

Reply:
My general rule of thumb for naming things is that they should give a hint about what something **does**, rather than describe what it **is**. So a datastructure containing customer information implemented as a  linked list should be named "customer_info" and not "customer_linked_list". Here `proc-macros` tells the reader what it is, but gives no hint about what it does.

Let's yak shave this name and come up with something better before `0.2`. Not a blocker for this PR.